content/articles/understanding-how-cluster-marker-works/index.md

```diff
@@ -0,0 +1,239 @@
+### Understanding how cluster markers work in web mapping using leaflet.js
+  This is no more a news, as we are all aware that big data is the keyword of the 21st century, and It's shocking to realize that only a few people understand that a large part of data is categorized as geospatial. Nowadays, nearly all of the gadgets we use are satellite-based systems like GPS. Today, the simplest case of displaying data/marker on a map with an acceptable user experience may be a real challenge especially for the user as they might find it difficult to understand and to analyze data at a particular zoom level. For instance, a platform that has users with hundreds of markers on the map, when you zoom out the map, these markers all overlap and make the map appear messy and crowdy. To improve this kind of map, it is recommended to use the marker clustering method.
+
+### Prerequisites
+To Understand this tutorial, it is assumed that the reader knows the below technologies:
+- JavaScript
+- HTML/CSS
+  
+### Goal
+At the end of this guide, we'll build a map with clustering features, and these include:
```

Comment:
You don't build the features at the end!! Do you build it from the start? In the end what should have the learner gain reading your article?